docs/cell.editing.md

```diff
@@ -1,12 +1,39 @@
-# qgrid 
-## markup
+# Cell editing
+To switch on cell editing specify edit-mode="cell" in q-grid:
+### Markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid columns="$ctrl.columns"
+               rows="$ctrl.rows"
+               edit-mode="cell">
+</q-grid>
 ```
-## script
-```javascript
-controller.$inject = ['$scope'];
-function controller($scope){
-	$scope.data = [];
-}
-```
\ No newline at end of file
+You can override default cell edit template for columns:
+```html
+<q-grid columns="$ctrl.columns"
+               rows="$ctrl.rows"
+               edit-mode="cell">
+    <q-grid:columns>
+        <q-grid:column key="name.first">
+            <q-grid:template for="edit">
+                 <input type="text"
+                     q-grid:focus
+                     style="border: 1px solid #f00;"
+                     ng-blur="$view.edit.cell.commit.execute($cell)"
```

Comment:
Can you please accent these two rows? It's the key for user to know, that `$view.edit.cell.value` is a model and that `$view.edit.cell.commit.execute($cell)` must be invoked to exit edit mode.